code/modules/admin/admin.dm

```diff
@@ -842,3 +842,83 @@
 		if(string)
 			message_admins("[string]")
 
+/datum/admins/proc/sendFax()
+	set category = "Special Verbs"
+	set name = "Send Fax"
+	set desc = "Sends a fax to this machine"
+	var/department = input("Choose a fax", "Fax") as null|anything in GLOB.all_departments
+	for(var/obj/machinery/photocopier/faxmachine/sendto in GLOB.all_faxes)
+		if(sendto.department == department)

```

Comment:
if departments can only have 1 fax machine then make this an associative list. You can do `P.destination = GLOB.all_faxes[department]` then.

Regardless, this should be an untyped loop since I assume GLOB.all_faxes only contains fax machines.